cfn-resources/project/docs/README.md

```diff
@@ -14,7 +14,12 @@ To declare this entity in your AWS CloudFormation template, use the following sy
     "Properties" : {
```

Comment:
For MongoDB::Atlas::Project description at top we should copy the Documentation definition as close as possible to prevent confusion. For Project it is  "Returns, adds, and edits collections of clusters and users in MongoDB Cloud." https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Projects

Reply:
It doesn't sound correct to me, still added as suggested.

Reply:
Thanks. you're welcome to open another doc PR to change definition in our OpenAPI docs :) this likely goes back to use case of almost all customers leveraging Atlas to create clusters and you can't create a cluster without first creating a Project

Reply:
It's not a very good definition, agree.  I think what it is trying to express is that a lot of settings that impact clusters and users are at the project level.  So from an organizational perspective this is true.

Reply:
Let's revisit with the dedicated Documentation PR for MongoDB::Atlas::Project as discussed. Thanks @govardhanpagidi !